client/data/settings/selectors.js

```diff
@@ -75,6 +75,12 @@ export const getAccountBusinessSupportAddressCity = ( state ) => {
 	return getSettings( state ).account_business_support_address.city || '';
 };
 
+export const getAccountBusinessSupportAddressState = ( state ) => {
+	if ( ! getSettings( state ).account_business_support_address ) return '';
```

Comment:
Let' make it a little bit more READABLE:

```javascript
let support address = getSettings( state ).account_business_support_address || {};
return address..state || '';
```

Reply:
Updated!